src/proof/alethe/alethe_printer.h

```diff
@@ -0,0 +1,63 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Hanna Lachnitt
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * The module for printing Alethe proof nodes
+ */
+
+#include "cvc5_private.h"
+
+#ifndef CVC4__PROOF__ALETHE_PROOF_PRINTER_H
+#define CVC4__PROOF__ALETHE_PROOF_PRINTER_H
+
+#include "proof/proof_node.h"
+
+namespace cvc5 {
+
+namespace proof {
+
+class AletheProofPrinter
+{
+ public:
+  AletheProofPrinter();
+  ~AletheProofPrinter() {}
+  /**
+   * This method prints a proof node that has been transformed into the Alethe
+   * proof format
+   *
+   * @param out The stream to write to
+   * @param pfn The proof node to be printed
+   */
+  void alethePrinter(std::ostream& out, std::shared_ptr<ProofNode> pfn);
+
+ private:
+  /** Used for printing the node after the initial anchor has been printed */
+  std::string alethePrinterInternal(std::ostream& out,
+                                    std::shared_ptr<ProofNode> pfn);
+  /** The current level of nesting, which increases if a subproof is entered */
+  int nested_level;
```

Comment:
You should use `size_t` for numerical values to be used to access indices of standard C++ containers.

Moreover all attributes of a class should have a "d_" at its prefixes and be in caml case. So the above should be `d_nestedLevel`.